README.md

```diff
@@ -1,17 +1,17 @@
-[![Build Status](https://img.shields.io/travis/seek-oss/eslint-config-sku/master.svg?style=flat-square)](http://travis-ci.org/seek-oss/eslint-config-sku) [![npm](https://img.shields.io/npm/v/eslint-config-sku.svg?style=flat-square)](https://www.npmjs.com/package/eslint-config-sku) [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release) [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg?style=flat-square)](http://commitizen.github.io/cz-cli/)
+[![Build Status](https://img.shields.io/travis/seek-oss/eslint-config-seek/master.svg?style=flat-square)](http://travis-ci.org/seek-oss/eslint-config-seek) [![npm](https://img.shields.io/npm/v/eslint-config-seek.svg?style=flat-square)](https://www.npmjs.com/package/eslint-config-seek) [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release) [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg?style=flat-square)](http://commitizen.github.io/cz-cli/)
 
 
-# eslint-config-sku
+# eslint-config-seek
 
-This package includes the shareable ESLint configuration used by [sku](https://github.com/seek-oss/sku).
+This package includes the shareable ESLint configuration used by SEEK.
```

Comment:
Our pattern with the other config repos has been linking `SEEK` to github.com/seek-oss